libfreerdp/core/settings.c

```diff
@@ -1763,8 +1763,6 @@ BOOL freerdp_settings_enforce_monitor_exists(rdpSettings* settings)
 		    freerdp_settings_get_pointer_array_writable(settings, FreeRDP_MonitorDefArray, 0);
 		if (!monitor)
 			return FALSE;
-		monitor->x = 0;
```

Comment:
wrong fix, the setting here is to enforce `RDP` requirements.
correct would be to map the `0/0` to native.